SDWebImage/SDWebImageManager.m

```diff
@@ -140,9 +140,7 @@ - (void)diskImageExistsForURL:(nullable NSURL *)url
         return operation;
     }
 
-    @synchronized (self.runningOperations) {
-        [self.runningOperations addObject:operation];
```

Comment:
...I think this should be `safelyAddOperationToRunning`. Not `safelyRemoveOperationFromRunning` :)